docs/guides/getting-started/index.html

```diff
@@ -97,10 +97,16 @@ <h2 id="step-2">Step 2: Install Polyfills</h2>
       </li>
       <li>Proceed to "Step 3: Choose a Layout".</li>
     </ol>
+    <p class="hxSubBody hxSubdued">
+      <hx-icon type="info-circle"></hx-icon>
+      Version 2.x (or later) is recommended, because it is more efficient
+      in loading required polyfill bundles.
+    </p>
     <hx-tabset class="beta-hxBound">
       <hx-tablist>
         <hx-tab>NPM</hx-tab>
         <hx-tab>Download</hx-tab>
+        <hx-tab>unpkg</hx-tab>
```

Comment:
Should we add a warning/callout that we don't recommend using `unpkg` in production?

Reply:
ok sounds good...zoom LGTM